src/test_storageiface.py

```diff
@@ -0,0 +1,139 @@
+'''
+test_storageiface.py
+
+Basic unit testing of the storage interfaces
+'''
+
+import unittest
+import logging
+import configparser
+
+class TestStorage(unittest.TestCase):
+  '''Simple tests for the storage layers of the WOPI server'''
+
+  def __init__(self, *args, **kwargs):
+    '''One-off initialization of the test environment: create mock logging and import the library'''
+    super(TestStorage, self).__init__(*args, **kwargs)
+    loghandler = logging.FileHandler('/tmp/wopiserver.log')
+    loghandler.setFormatter(logging.Formatter(fmt='%(asctime)s %(name)s[%(process)d] %(levelname)-8s %(message)s',
+                                              datefmt='%Y-%m-%dT%H:%M:%S'))
+    log = logging.getLogger('wopiserver.test')
+    log.addHandler(loghandler)
+    log.setLevel(logging.DEBUG)
+    # read the configuration
```

Comment:
I don't think this comment is needed, `config = configparser.ConfigParser()` is rather self explanatory.